internal/cmd/iam/command_groupmapping_update.go

```diff
@@ -0,0 +1,74 @@
+package iam
+
+import (
+	"github.com/spf13/cobra"
+
+	ssov2 "github.com/confluentinc/ccloud-sdk-go-v2/sso/v2"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+)
+
+const groupMappingNoOpUpdateErrorMsg = "one of `--description`, `--name`, or `--filter` must be set"
```

Comment:
Do we have a global format string that we can use for all of our update commands?

Reply:
I don't think we have one? A no-op error only makes sense when all the fields in a resource's `update` API call are optional, so we can just avoid an unnecessary API call, which isn't the case for a lot of resources' `update` commands.

From a initial look-around, it looks like the `group-mapping, provider, pool` resources involved in this PR could use a global format string for an `update` no-op error msg, and also `pipeline`, `kafka cluster`, `kafka quota`, and `context`. Since there are quite a few resources that could use this and this PR is already getting quite large, I think I'll just make a ticket and do it in another PR.